tests/antithesis/README.md

```diff
@@ -0,0 +1,95 @@
+This directory enables integration of Antithesis with etcd. There are 4 containers running in this system: 3 that make up an etcd cluster (etcd0, etcd1, etcd2) and one that "[makes the system go](https://antithesis.com/docs/getting_started/basic_test_hookup/)" (client).
+
+## Quickstart
+
+### 1. Build and Tag the Docker Image
+
+Run this command from the `antithesis/test-template` directory:
+
+```bash
+docker build . -f Dockerfile.client -t etcd-client:latest
+```
+
+### 2. (Optional) Check the Image Locally
+
+You can verify your new image is built:
+
+```bash
+docker images | grep etcd-client
+```
+
+It should show something like:
+
+```
+etcd-client        latest    <IMAGE_ID>    <DATE>
+```
+
+### 3. Use in Docker Compose
+
+If you have a `docker-compose.yml` that defines your etcd cluster plus this client, make sure the `client` service references `etcd-client:latest`, for example:
+
+```yaml
+services:
+  etcd0:
+    image: docker.io/bitnami/etcd:3.5
+    environment:  # ...
+  etcd1:
+    image: quay.io/coreos/etcd:v3.5.0
```

Comment:
```suggestion
    image: gcr.io/etcd-development/etcd:v3.5.21
```

Reply:
See https://github.com/etcd-io/etcd/pull/19739#discussion_r2041688666 (sorry I think @serathius  and I started reviewing at the same time...). His comment takes precedence, so dropping mine!